### PR TITLE
multiple edits - improved S5D9 schematic

### PR DESCRIPTION
**Pull request by jcasman**

### CODE REVIEW

**Review by codetricity — COMMENTED** [review]
big improvement on the schematic picture. wonderful

I also like the smaller picture of the board with the sticker on it.

File: docs/monitoring/starting.md [hunk 1]
```diff
@@ -15,45 +15,50 @@
...
 ![](img/thing-box.png)
 
-At this point, you should see sensor data from your board streaming to the dashboard at an update rate of approximately once per second.
+At this point, after just a few seconds, you should see sensor data from your board streaming to the dashboard at an update rate of approximately once per second.
 
 > NOTE: if you don't see any sensor data after 3 minutes, unset and then set your Board ID again.
 
 ![](img/unset.png)
 
+Terminology: The Renesas Data Monitoring Dashboard is also referred to as *Freeboard*
```
Comment (codetricity) [inline docs/monitoring/starting.md:60]: Nice. Improved flow by pushing "Terminology" down. We might also want to explain what a Dweet is in the other document.

**Comment by jcasman** [comment]
Actually learned a lot about the board by spending time improving the schematic. One Grove connector is IC2 and the other is UART

File: docs/monitoring/starting.md (hunk 1 above)
Comment (camera360developers) [reply docs/monitoring/starting.md:60]: I will add info about Dweets, good idea

Comment (codetricity) [reply docs/monitoring/starting.md:60]: do it, my man.

**Pull request merged**